src/components/html/EcAppsMarket.html

```diff
@@ -1,49 +1,72 @@
 <div class="ec-apps-market">
-  <a-card
-    class="mb-4"
-    :bodyStyle="{ paddingBottom: 0 }"
-  >
+  <div class="card pt-4 pl-4">
     <h2>{{ i19applications }}</h2>
-    <a-tabs
-      :tabBarStyle="{ margin: 0 }"
+    <ul 
+      class="nav nav-tabs" 
       v-model="activeTabKey"
     >
-      <a-tab-pane :tab="i19availableApps" key="market"/>
-      <a-tab-pane :tab="i19yourInstalledApps" key="installed"/>
-    </a-tabs>
-  </a-card>
-
-  <fade-transition :duration="{ enter: 500, leave: 0 }">
-    <div v-if="!loading" class="pt-2">
-      <a-alert
-        v-if="loadError"
-        key="error"
-        :message="errorMessage"
-        type="warning"
-        closable
-        @close="updateTabContent"
-        :closeText="i19tryAgain"
-      />
+      <li class="nav-item">
+        <a 
+          class="nav-link" 
+          :class="{ active: activeTabKey === 'market' }" 
+          role="button" 
+          key="market" 
+          @click="switchTabs('market')"
+        >
+          {{ i19availableApps }}
+        </a>
+      </li>
+      <li class="nav-item">
+        <a 
+          class="nav-link" 
+          :class="{ active: activeTabKey === 'installed' }" 
+          role="button" 
+          key="installed" 
+          @click="switchTabs('installed')"
+        >
+          {{i19yourInstalledApps}}
+        </a>
+      </li>
```

Comment:
Na sua implementação não precisamos mais do `key` no `<a>`, também não vi vantagem em criar o método `switchTabs` mas sem problemas com isto também, vou só segurir uma implementação mais DRY:

```suggestion
      <li
        v-for="(label, key) in tabs"
        class="nav-item"
        :key="key"
      >
        <a 
          class="nav-link" 
          :class="{ active: activeTabKey === key }" 
          role="button" 
          @click="activeTabKey = key"
        >
          {{ label }}
        </a>
      </li>
```

Neste caso no JS do componente você precisaria de um _computed_ `tabs`:

```js
tabs () {
  return {
    market: this.i19availableApps,
    installed: this.i19yourInstalledApps
  }
}
```